massa-consensus-worker/src/tests/tools.rs

```diff
@@ -125,6 +125,7 @@ pub fn create_block_with_merkle_root(
     .unwrap()
 }
 
+#[allow(clippy::borrowed_box)]
```

Comment:
Why can't we just use a Box here?

Reply:
It makes all the calls fails and I want to change it when I will re-rewrite the tests of consensus with the new universe don't wanna lost time on this now.